### PR TITLE
Role Management UI Fixes.

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx [hunk 1]
```diff
@@ -47,6 +48,17 @@ export const RoleBasics: FunctionComponent<RoleBasicProps> = (props: RoleBasicPr
         isAddGroup
     } = props;
 
+    const [ isValidRoleName, setIsValidRoleName ] = useState<boolean>(true);
+    const [ updatedRoleName, setUpdatedRoleName ] = useState<string>(initialValues?.roleName);
+
+    /**
+     * Triggers when updatedRoleName is changed
```
Comment (inthirakumaaran) [inline apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx:55]:
```suggestion
     * Triggers when updatedRoleName is changed.
```

File: apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx [hunk 2]
```diff
@@ -66,6 +78,35 @@ export const RoleBasics: FunctionComponent<RoleBasicProps> = (props: RoleBasicPr
...
+        searchRoleList(searchData)
+            .then((response) => {
+                setIsValidRoleName(response?.data?.totalResults === 0);
+            });
+    };
+
+    /**
+     * The following function handles the change of the username.
+     *
+     * @param values
```
Comment (inthirakumaaran) [inline apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx:104]: Add the parameter comment

File: apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx (hunk 1 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx]: fixed with 49627de

File: apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx (hunk 2 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/src/components/roles/create-role-wizard/role-basics.tsx]: fixed with 49627de

**Pull request merged**